web/app/js/components/Tap.jsx

```diff
@@ -131,6 +129,60 @@ class Tap extends React.Component {
     }, {});
   }
 
+  parseTapResult = data => {
+    let d = JSON.parse(data);
+
+    switch (d.proxyDirection) {
+      case "INBOUND":
+        d.tls = _.get(d, "sourceMeta.labels.tls", "");
+        break;
+      case "OUTBOUND":
+        d.tls = _.get(d, "destinationMeta.labels.tls", "");
+        break;
+      default:
+        // too old for TLS
+    }
+
+    if (_.isNil(d.http)) {
+      this.setState({ error: "Undefined request type"});
```

Comment:
Just a heads up that this conflicts with #1395, so would be good to bear in mind when you're reviewing that branch.